server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -1156,11 +1156,11 @@ public TSStatus insertRecords(TSInsertRecordsReq req) {
         TSStatus status = checkAuthority(plan, req.getSessionId());
         if (status != null) {
           insertRowsPlan.getResults().put(i, status);
-          allSuccess = false;
```

Comment:
Even if the check fails, the insertRowsPlan will continue.

Reply:
it will not be executed, this is done in the insert(InsertRowsPlan plan)  method in PlanExecutor.